### PR TITLE
181163219_show_geo_data_in_api

### PR DESCRIPTION
**Pull request by radek-polcode**
#### What's this PR do?
- Adds latitude and longitude to api response + update the docs

#### How should this be manually tested?
- Use postmant or curl, request validators list or single validator (make sure it has ip record connected) and check if latitude and longitude are present

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181163219

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: test/controllers/api/v1/api_controller_test.rb [hunk 1]
```diff
@@ -149,7 +149,10 @@ def setup
     assert_nil validator_with_all_data["ping_time"]
 
     # IP
```
Comment (psuchta) [inline test/controllers/api/v1/api_controller_test.rb:151]: Please add a test for checking if `latitude` and `longitude` return values when they are not nil

**Review by psuchta — CHANGES_REQUESTED** [review]
Please add a test for checking if `latitude` and `longitude` return values when they are not nil

**Review by psuchta — APPROVED** [review]
